Demo/DemoApp/DemoApp.xcodeproj/project.pbxproj

```diff
@@ -488,6 +489,7 @@
 			buildSettings = {
 				ASSETCATALOG_COMPILER_APPICON_NAME = AppIcon;
 				CODE_SIGN_STYLE = Automatic;
+				DEVELOPMENT_TEAM = B6333T4PXQ;
```

Comment:
What team are you setting this to? Is this needed?

Reply:
I set it to something so I can replace it in the build like this:

> sed -i '' 's/DEVELOPMENT_TEAM = B6333T4PXQ;/DEVELOPMENT_TEAM = 72SA8V3WYL;/g' "$PROJ"

Is there a better value to set it to? If it's absent then it's more painful to update via a shell script.

Reply:
Looks like empty string works.